dll/win32/netapi32/group_new.c

```diff
@@ -379,9 +379,11 @@ NetGroupAdd(
     if (ApiStatus == NERR_Success)
     {
         ERR("OpenGroupByName: Group %wZ already exists!\n", &GroupName);
+        ApiStatus = ERROR_GROUP_EXISTS;
 
         SamCloseHandle(GroupHandle);
-        ApiStatus = ERROR_GROUP_EXISTS;
+        GroupHandle = NULL;
```

Comment:
`SamCloseHandle` already invalidates the handle object for you.
See https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/samsrv/samrpc.c;hb=9f3ed60ea4779a8848c3a78d004f415520669832#l160

Reply:
I am not sure what your point is. You linked to `SamrCloseHandle()`, which is not the same function...

Reply:
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/samlib/samlib.c;hb=6d441f7d65f57c0f5b6da59ae851640a6e787875#l495

Clearly do I have to babysit you on grep'ing stuff now, huh? :trollface: